### PR TITLE
DDF-1877 Updates Admin UI to show human readable names for MSF configs

### PR DESCRIPTION
**Pull request by andrewkfiedler**
- Adds a method to the Service model that returns the human readable name for a configuration.
 - Updates the template for configurationRow to utilize the displayName property.
 - Updates the service view to call the new model method to get a human readable name, and to merge this data into the object that is given to the template (that way it can access the displayName property).

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/540" target="_blank">ReviewNinja

**Comment by andrewkfiedler** [comment]
@djblue @oconnormi @rzwiefel

File: platform/admin/ui/src/main/webapp/js/views/configuration/Service.view.js [hunk 1]
```diff
@@ -62,6 +62,9 @@ define([
                     configuration.close();
                 });
             }
+        },
+        serializeData: function () {
```
Comment (lessarderic) [inline platform/admin/ui/src/main/webapp/js/views/configuration/Service.view.js]: Do we need to serialize the `id` here? Can't we just get it back from the model when needed?

File: platform/admin/ui/src/main/webapp/js/models/Service.js [hunk 2]
```diff
@@ -212,6 +215,14 @@ define(['backbone', 'jquery','backboneassociations'],function (Backbone, $) {
                     return defaults;
                 }, {}));
             return this;
+        },
+        getConfigurationDisplayName: function () {
+            var displayName = this.get('id');
+            var properties = this.get('properties');
+            if (isServiceFactory(properties)) {
+                displayName = properties.get('name') || properties.get('shortname') || properties.get('id') || displayName;
```
Comment (lessarderic) [inline platform/admin/ui/src/main/webapp/js/models/Service.js]: I assume `model.id` is different from `properties.get('id')`? If not, we should be able to remove `|| properties.get('id')` from the chain.

Comment (andrewkfiedler) [inline platform/admin/ui/src/main/webapp/js/models/Service.js]: It's different.

File: platform/admin/ui/src/main/webapp/js/views/configuration/Service.view.js (hunk 1 above)
Comment (andrewkfiedler) [inline platform/admin/ui/src/main/webapp/js/views/configuration/Service.view.js]: I'm not sure what you mean. This is to add the displayName into the object that is being given to the template.

Comment (lessarderic) [inline platform/admin/ui/src/main/webapp/js/views/configuration/Service.view.js]: Nvm then. Stupid question :confused:

**Comment by djblue** [comment]
LGTM :+1:

**Comment by rzwiefel** [comment]
:shipit: :+1:

**Comment by oconnormi** [comment]
:+1:

**Comment by rzwiefel** [comment]
Built and started up. Everything worked reaaaal good. :+1:
:shipit:

**Pull request merged**